src/DssSpell.sol

```diff
@@ -59,167 +56,125 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
-    uint256 constant ZERO_PCT           = 1000000000000000000000000000;
-    uint256 constant ONE_PCT            = 1000000000315522921573372069;
-    uint256 constant TWO_PCT            = 1000000000627937192491029810;
-    uint256 constant THREE_PCT          = 1000000000937303470807876289;
-    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
-    uint256 constant FOUR_PCT           = 1000000001243680656318820312;
-    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
-    uint256 constant FIVE_PCT           = 1000000001547125957863212448;
-    uint256 constant TEN_PCT            = 1000000003022265980097387650;
+    // uint256 constant ZERO_PCT           = 1000000000000000000000000000;
+    // uint256 constant ONE_PCT            = 1000000000315522921573372069;
+    // uint256 constant TWO_PCT            = 1000000000627937192491029810;
+    // uint256 constant THREE_PCT          = 1000000000937303470807876289;
+    // uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
+    // uint256 constant FOUR_PCT           = 1000000001243680656318820312;
+    // uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
+    // uint256 constant FIVE_PCT           = 1000000001547125957863212448;
+    // uint256 constant TEN_PCT            = 1000000003022265980097387650;
```

Comment:
I would cleanup the Rates commented as well, including the rates comment block above.

Reply:
why? Better leave them there for simplifying verification of future spells, no?

Reply:
yeah, it's ok I guess

Reply:
@gbalabasquer shares your opinion, so removing

Reply:
Maybe leave one as an example, but we usually clear these out when we're not looking at them.